src/include/connection.h

```diff
@@ -42,6 +42,44 @@ struct __wt_process {
 };
 extern WT_PROCESS __wt_process;
 
+#define WT_BKG_COMPACT_INSERT(conn, dsrc_stat, bucket)                                        \
```

Comment:
Could you use `BACKGROUND` instead of `BKG`? It always takes me a while to assimilate new acronyms..

Also: Did you think about implementing these as functions instead of macros?

Reply:
I kept this in line with how we handle other lists, but I do prefer functions so if that's acceptable I'll change it.

Reply:
Fixed in https://github.com/wiredtiger/wiredtiger/pull/9541/commits/2dbabe0130c1b0775f5b4c493c6e84747ccb2296.